x-pack/legacy/plugins/monitoring/public/components/alerts/alerts.js

```diff
@@ -5,7 +5,7 @@
  */
```

Comment:
Why are these files still in `x-pack/legacy/plugins`? What do we need to do to move them to `x-pack/plugins`?

Reply:
@chrisronline I wanted to avoid merge conflicts (and loss of any new changes). I moved everything outside legacy folder, but now this PR should be merged before any other monitoring PR